mlos_bench/mlos_bench/launcher.py

```diff
@@ -59,6 +61,7 @@ def __init__(self, description: str):
             config = {}
 
         log_level = args.log_level or config.get("log_level", _LOG_LEVEL)
+        log_level = logging.getLevelName(log_level)
```

Comment:
We need to double check if this works. First of all, `_LOG_LEVEL` declared above is still an `int`. Second, `logging.getLevelName()` seem to have a bit weird behavior:
```python
In [7]: logging.getLevelName(logging.INFO)
Out[7]: 'INFO'

In [8]: logging.getLevelName("INFO")
Out[8]: 20
```
😮 

I just hope that `.setLevel()` is equally weird (ok, "flexible") and it all works out at the end 😄

Reply:
Besides, it might be a good idea to allow *both* `int` *and* enum log levels - it is ok to have a log level of, e.g., 15 for messages which importance is somewhere between debug and info

Reply:
I did some limited tests and it appears to work.  And will properly throw an error if you give it a garbage value like `--log-level foo`
I'm not sure why we'd want to support in-between log levels though - they're pretty non-standard and not very intuitive.

Reply:
I have another small fix for this pending but github won't let me push atm (https://www.githubstatus.com/ is showing issues).